src/main/java/RedisSessionManagerT8.java

```diff
@@ -0,0 +1,309 @@
+    }
+
+    @Override
+    public void remove(Session session) {
+
+        remove(session, false);
+    }
+
+    @Override
+    public void remove(Session session, boolean update) {
+        //Redis will remove the attributes
+    }
+
+    @Override
+    public void processExpires() {
+        // We are going to use Redis's ability to expire keys for session expiration.
+        // Do nothing.
+    }
+
+    //------------------------------------------------------------------------------------------------------------------
+    // END Sessions
+    //------------------------------------------------------------------------------------------------------------------
+
+
+    //------------------------------------------------------------------------------------------------------------------
+    // START Utils
+    //------------------------------------------------------------------------------------------------------------------
+
+    /**
+     * Get session from metadata and attributes
+     * @param metadata
+     * @param attributes
+     * @return
+     */
+    private RedisSessionT8 getSession(Hashtable<String, Object> metadata, Hashtable<String, Object> attributes){
+        RedisSessionT8 session = (RedisSessionT8) createEmptySession();
+        session.setValid(Boolean.valueOf((String) metadata.get(METADATA_VALID)));
+
+        try {
+            session.setCreationTime(DateFormatUtils.ISO_DATE_FORMAT.parse(((String) metadata.get(METADATA_CREATION_TIME))).getTime());
+        } catch (ParseException e) {
+            log.error("Error - Context: getSession. Description: " + e.getMessage());
+        }
+
+        try {
+            session.setLastAccessedTime(DateFormatUtils.ISO_DATE_FORMAT.parse(((String) metadata.get(METADATA_LAST_ACCESS_TIME))).getTime());
+        } catch (ParseException e) {
+            log.error("Error - Context: getSession. Description: " + e.getMessage());
+        }
+
+        session.setMaxInactiveInterval(Integer.valueOf((String) metadata.get(METADATA_MAX_INACTIVE_INTERVAL)));
+
+        for (Enumeration<String> enumerator = attributes.keys(); enumerator.hasMoreElements();) {
+            String key = enumerator.nextElement();
+            session.setAttribute(key, attributes.get(key));
+        }
+
+        return session;
+    }
+
+
+    /**
+     * Get metadata from a Session
+     * @param session
+     * @return
+     */
+    private Hashtable<String, Object> getMetadata(Session session){
+        Hashtable<String, Object> metadata = new Hashtable<>();
+        metadata.put(METADATA_VALID, String.valueOf(session.isValid()));
+        metadata.put(METADATA_CREATION_TIME, DateFormatUtils.ISO_DATE_FORMAT.format(new Date(session.getCreationTime())));
+        metadata.put(METADATA_LAST_ACCESS_TIME, DateFormatUtils.ISO_DATE_FORMAT.format(new Date(session.getLastAccessedTime())));
+        metadata.put(METADATA_MAX_INACTIVE_INTERVAL, String.valueOf(session.getMaxInactiveInterval()));
+        return metadata;
```

Comment:
Why are you using a `Hashtable` here instead of a `Map`?  Or, better yet, an object?